src/SqlPersistence/InstallerFeature.cs

```diff
@@ -16,7 +16,11 @@ protected override void Setup(FeatureConfigurationContext context)
             return;
         }
 
-        settings.ConnectionBuilder = context.Settings.GetConnectionBuilder();
+        settings.ConnectionBuilder = storageType =>
```

Comment:
Could be `settings.ConnectionBuilder = storageType => context.Settings.GetConnectionBuilder(storageType)();` or you keep it that way for readability?

If that was the explicit intention then ignore comment.

Reply:
ping